package.json

```diff
@@ -245,7 +245,7 @@
 		"postinstall": "npm run build-packages",
 		"lint": "run-s -s lint:*",
 		"lint:config-defaults": "node server/config/validate-config-keys.js",
-		"lint:css": "stylelint \"client/**/*.scss\" --syntax scss",
+		"lint:css": "stylelint \"**/*.scss\" --syntax scss",
```

Comment:
Seems to align with our precommit:

https://github.com/Automattic/wp-calypso/blob/b2440a001d1694f35d0444a7bb670aec0ed2b32a/bin/pre-commit-hook.js#L30